code/modules/mob/living/carbon/xenomorph/abilities.dm

```diff
@@ -815,6 +815,28 @@
 	X.recurring_injection(A, /datum/reagent/toxin/xeno_neurotoxin, XENO_NEURO_CHANNEL_TIME, XENO_NEURO_AMOUNT_RECURRING)
 
 
+//Panther Neurotox Sting
+/datum/action/xeno_action/activable/neurotox_sting/panther
+	name = "Panther Neurotoxin Sting"
+	mechanics_text = "A channeled melee attack that injects the target with neurotoxin over a few seconds, temporarily stunning them."
+	ability_name = "panther neurotoxin sting"
+	cooldown_timer = 50 SECONDS
+	plasma_cost = 60
+	keybind_signal = COMSIG_XENOABILITY_NEUROTOX_STING
```

Comment:
This means the string will share a keybind.
I personally don't mind, just making sure its intended.

Reply:
I rather doubt it'd cause any issues.

Reply:
Just FYI, the sting won't stun. It'll keep them from regenning stamina for about 45 seconds if you get the whole thing off.

Reply:
Give it its own signal.